editoast/Cargo.toml

```
@@ -51,6 +52,8 @@ editoast_derive = { path = "./editoast_derive" }
 mvt = "0.8.1"
 pointy = "0.4.0"
 futures = "0.3"
+postgis_diesel = { version = "2.1.0", features = ["serde"] }
+uuid = { version = "1.3.0" }
```

Diesel fix its uuid dependency like so:
```
uuid = { version = ">=0.7.0, <2.0.0", optional = true }
```


```suggestion
uuid = { version = "1" }
```